libpamc/Makefile.am

```diff
@@ -22,3 +22,8 @@ endif
 lib_LTLIBRARIES = libpamc.la
 
 libpamc_la_SOURCES = pamc_client.c pamc_converse.c pamc_load.c
+
+# Pkg-config script.
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = pam.pc
+DISTCLEANFILES = $(pkgconfig_DATA)
```

Comment:
Since files created by `AC_CONFIG_FILES` are removed by `make distclean` automatically, no `DISTCLEANFILES` is needed.